src/modality-specific-files/eye-tracking.md

```diff
@@ -0,0 +1,233 @@
+
+Generic fields MUST be present:
+
+{{ MACROS___make_metadata_table(
+   {
+      "TaskName": ("REQUIRED", "A RECOMMENDED convention is to name resting state task using labels beginning with `rest`."),
+   }
+) }}
+
+SHOULD be present:
+
+For consistency between studies and institutions, we encourage users to extract the values of these fields
+from the actual raw data. Whenever possible, please avoid using ad hoc wording.
+
+{{ MACROS___make_metadata_table(
+   {
+      "InstitutionName": "RECOMMENDED",
+      "InstitutionAddress": "RECOMMENDED",
+      "Manufacturer": (
+         "RECOMMENDED",
+         "Manufacturer of the eye-tracking system (for example "
+         '`"SR-Research"`, `"Tobii"`, `"SMI"`, `"Gazepoint"`, '
+         '`"Pupil Labs"`, `"Custom built"`, `"Other"`) '
+      ),
+      "ManufacturersModelName": (
+         "RECOMMENDED",
+         "Manufacturer's designation of the eye-tracker model (for example"
+         '`"Eye-link 1000"`).'
+         ),
+      "SoftwareVersions": "RECOMMENDED",
+      "TaskDescription": "RECOMMENDED",
+      "Instructions": (
+         "RECOMMENDED",
+         "Text of the instructions given to participants before the experiment."),
+      "CogAtlasID": "RECOMMENDED",
+      "CogPOID": "RECOMMENDED",
+      "DeviceSerialNumber": "RECOMMENDED",
+   }
+) }}
+
+Specific eye-tracking fields MUST be present:
+
+{{ MACROS___make_metadata_table(
+   {
+      "SamplingFrequency__eyetrack": "REQUIRED",
+      "SampleCoordinateUnit": "REQUIRED",
+      "SampleCoordinateSystem": "REQUIRED",
+      "EnvironmentCoordinates": "REQUIRED",
+      "ScreenSize": "REQUIRED",
+      "ScreenResolution": "REQUIRED",
+      "ScreenDistance": (
+         "REQUIRED",
+         "Screen distance in cm (for example 60 for a screen distance of 60 cm), "
+         'if no screen use "n/a". '
+         "For MRI, it corresponds to the distance between the head-coil mirror "
+         "to the projection screen for example."),
+   }
+) }}
+
+Specific eye-tracking fields SHOULD be present:
+
+{{ MACROS___make_metadata_table(
+   {
+      "IncludedEyeMovementEvents": "RECOMMENDED",
+      "DetectionAlgorithm": "RECOMMENDED",
+      "DetectionAlgorithmSettings": "RECOMMENDED",
+      "StartMessage": "RECOMMENDED",
+      "EndMessage": "RECOMMENDED",
+      "KeyPressMessage": "RECOMMENDED",
+      "CalibrationType": "RECOMMENDED",
+      "CalibrationUnit": "RECOMMENDED",
+      "CalibrationPosition": "RECOMMENDED",
+      "MaximalCalibrationError": "RECOMMENDED",
+      "AverageCalibrationError": "RECOMMENDED",
+      "CalibrationList": "RECOMMENDED",
+      "RecordedEye": "RECOMMENDED",
+      "EyeCameraSettings": "RECOMMENDED",
+      "FeatureDetectionSettings": "RECOMMENDED",
+      "GazeMappingSettings": "RECOMMENDED",
+      "RawDataFilters": "RECOMMENDED",
+      "ScreenRefreshRate": "RECOMMENDED",
+      "AOIDefinition": "RECOMMENDED",
+      "PupilFitMethod": "RECOMMENDED",
+   }
+) }}
```

Comment:
I can be wrong. @tsalo or some other BIDS schema guru could set me straight: I feel that this is duplicating information which is (should be) present in the schema and not explicitly listed here in addition.  Looking at similar section in microscopy: https://github.com/bids-standard/bids-specification/blob/master/src/modality-specific-files/microscopy.md#microscopy-metadata-sidecar-json (rendered [here](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/10-microscopy.html#microscopy-metadata-sidecar-json)) there seems to be `MACROS___make_sidecar_table`  which might be the one to use here to somehow automagically pick up all the relevant fields.

Reply:
I made everythin in `MACROS___make_sidecar_table` format in the new version to be released soon.